### ISSUE TITLE
set_default_value() with 'set' field results with error

### ISSUE THREAD

**Issue by dameer** [htmlburger/carbon-fields#447]
For some reason `set_default_value()` with 'set' field ends up with Error/Notice:

`Array to string conversion in ...../wp-content/themes/my_theme/carbon-fields/vendor/htmlburger/carbon-fields/core/Field/Field.php on line 605.`

Field definition:

```
		Field::make( 'set', 'events-cat-item-display', esc_attr__( 'What to show in loop?', 'events' ) )
		->set_options( array(
			'featured-image' => esc_attr__( 'Featured Image', 'events' ), 
			'title' => esc_attr__( 'Title', 'events' ), 
			'excerpt' => esc_attr__( 'Excerpt', 'events' ), 
			'event_date' => esc_attr__( 'Event date', 'events' ), 
			'event_location' => esc_attr__( 'Location', 'events' ), 
			'event_book' => esc_attr__( 'Add to GoogleCalendar button', 'events' ), 
		) )
		->set_default_value( array( 'featured-image', 'title', 'excerpt', 'event_date', 'event_location', 'event_book' ) )
		->set_help_text( esc_attr__( 'Show or hide certain elements for posts in loop.', 'events' ) ),
```
Any ideas why? I'm using Carbon Fields 2.1.0 as a library with my WP theme.

**Comment by dameer** [comment]
...I guess `strval($default_value)` can't handle array as default value but only string?

```
	/**
	 * Set default field value
	 *
	 * @param  mixed $default_value
	 * @return Field $this
	 */
	public function set_default_value( $default_value ) {
		$this->default_value = strval($default_value); /* Should check for value type first then return */
		return $this;
	}
```

**Comment by atanas-angelov-dev** [comment]
Hi @dameer ,

This has already been fixed in 2.1.1 - please make sure you are on the latest version.
